.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,7 +7,15 @@ Describe your proposed changes here.
 - [ ] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
 - [ ] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
 - [ ] Rebased/mergeable
+- [ ] Root cause description
+   * Describe the root cause of the bug. 
+   * Did the fix introduce the change in behavior.
```

Comment:
Can you add a question mark (?) at the end of the above statement (instead of a period)? Also, I don't understand the question. Are you asking, "Was there a code change that introduced this change in behavior?" or "Is this fix introducing a change in behavior?"

Reply:
It is about the fix. If fix introduces the change in behavior, we should know about this.

Reply:
updated